courses/free-courses-en.md

```diff
@@ -381,7 +381,7 @@
 ### Docker
 
 * [Docker](https://www.youtube.com/playlist?list=PLhW3qG5bs-L99pQsZ74f-LC-tOEsBp2rK) - Raghav Pal
-* [Docker Tutorial](https://www.tutorialspoint.com/docker/index.htm) - TutorialsPoint
+* [Docker Tutorial](https://www.tutorialspoint.com/docker/) - Tutorials Point (HTML, PDF)
```

Comment:
```suggestion
* [Docker Tutorial](https://www.tutorialspoint.com/docker/) - Tutorials Point (HTML, [PDF](https://www.tutorialspoint.com/docker/docker_tutorial.pdf))
```

Reply:
See https://github.com/EbookFoundation/free-programming-books/pull/6799#discussion_r836791385 for resolution